tests/testsuite/build.rs

```diff
@@ -103,14 +103,12 @@ fn cargo_fail_with_no_stderr() {
 fn cargo_compile_incremental() {
     let p = project()
         .file("Cargo.toml", &basic_bin_manifest("foo"))
-        .file("src/foo.rs", &main_file(r#""i am foo""#, &[]))
+        .file("src/main.rs", &main_file(r#""i am foo""#, &[]))
         .build();
 
     p.cargo("build -v")
         .env("CARGO_INCREMENTAL", "1")
         .with_stderr_data(str![[r#"
-[WARNING] path `src/foo.rs` was erroneously implicitly accepted for binary `foo`,
```

Comment:
We already have dedicated tests for this. No need to assert this warning message here.

Reply:
Thank you for updating the standards of the code under test to current best practices!